src/app/core/database/pouch-database.ts

```diff
@@ -27,6 +28,13 @@ import { LoggingService } from "../logging/logging.service";
  * should be implemented in the abstract {@link Database}.
  */
 export class PouchDatabase extends Database {
+  static createWithInBrowserDB(
```

Comment:
couldn't this just be named `create()` for both classes as the type is implied by the class name already?

Reply:
I know moved both static methods to `PouchDatabase`. Now the distinctive name makes sense.

Reply:
looks good :+1: 

Maybe we should call this `createWithIndexedDB`